test/e2e/flowaggregator_test.go

```diff
@@ -574,6 +563,26 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
 }
 
+func checkRecordsForToExternalFlows(t *testing.T, data *TestData, srcPodName string, srcIP string, dstIP string) {
+	timeStart := time.Now()
+	stdout, stderr, err := data.runCommandFromPod(testNamespace, srcPodName, busyboxContainerName, []string{"wget", "-O-", dstIP})
+	require.NoError(t, err, fmt.Sprintf("Error when running wget command: %v - stdout: %s - stderr: %s", err, stdout, stderr))
```

Comment:
`require.NoError` itself supports formatted message, and `err` would be duplicated in the output.
```
require.NoError(t, err, "Error when running wget command, stdout: %s, stderr: %s", stdout, stderr)
```

Reply:
Got it now, thanks for explaining this.